import_musicbrainz_row_ids.py

```diff
@@ -0,0 +1,41 @@
+from listenbrainz import config, db
+from brainzutils import musicbrainz_db
+
+import listenbrainz.db.user as db_user
+import sqlalchemy
+
+def import_musicbrainz_rows(musicbrainz_db_uri):
+    musicbrainz_db.init_db_engine(musicbrainz_db_uri)
+    db.init_db_connection(config.SQLALCHEMY_DATABASE_URI)
+    users = db_user.get_all_users()
+    with musicbrainz_db.engine.connect() as mb_connection:
+        for user in users:
+            name = user['musicbrainz_id']
+            result = mb_connection.execute(sqlalchemy.text("""
+                    SELECT id
+                      FROM editor
+                     WHERE LOWER(name) = LOWER(:name)
+                """), {
+                    'name': name,
+                })
+            musicbrainz_row_id = None
+            if result.rowcount > 0:
+                musicbrainz_row_id = result.fetchone()['id']
+            else:
+                print('No user with specified username in the MusicBrainz db: %s' % name)
+                continue
+
+            with db.engine.connect() as connection:
```

Comment:
If LB has a lot of users, it could be beneficial to create this connection outside the loop.